src/main/scala/coreplex/Coreplex.scala

```diff
@@ -65,21 +80,82 @@ abstract class Coreplex(implicit val p: Parameters, implicit val c: CoreplexConf
 }
 
 class DefaultCoreplex(tp: Parameters, tc: CoreplexConfig) extends Coreplex()(tp, tc) {
+  class TileWrapper(tileBuilder: (Bool, Parameters) => Tile, _clock: Clock, _reset: Bool)
+      (implicit p: Parameters) extends Module(Some(_clock), Some(_reset)) {
+
+    val wrappedTile = tileBuilder(reset, p)
+    val io = new Bundle {
+      val tile = wrappedTile.io.cloneType
+      val ext_clock = Clock(INPUT)
+      val ext_reset = Bool(INPUT)
+    }
+
+    io.tile.uncached <> wrappedTile.io.uncached.map(AsyncClientUncachedTileLinkTo(io.ext_clock, io.ext_reset, _))
+    io.tile.cached <> wrappedTile.io.cached.map(AsyncClientTileLinkTo(io.ext_clock, io.ext_reset, _))
+    wrappedTile.io.interrupts := LevelFrom(io.ext_clock, io.tile.interrupts)
```

Comment:
Expectation at ISA level is that FENCE suffices.  Doesn't matter how this is accomplished... if the mtimecmp write response takes at least as long to propagate as the interrupt itself, then it's correct automatically.  The current implementation happens to have that property, provided the same clock crossing latencies for the write response and the interrupt.  No need for a handshake in this case.